connector/oidc/oidc.go

```diff
@@ -87,6 +91,18 @@ type connectorData struct {
 	RefreshToken []byte
 }
 
+var managedAuthParams = []string{
```

Comment:
I'm a little concerned that this list is the only thing that protects parameters from being overwritten down below. Any chance we could make it so that parameters don't get overwritten? (Not sure it's easy or feasible at all, but it's worth a check and adding a comment here if it's not)